src/components/ActionCaption/ActionCaption.vue

```diff
@@ -61,5 +61,8 @@ export default {
 		pointer-events: none;
 		margin-left: 12px;
 		padding-right: 14px;
+		height: var(--clickable-area);
```

Comment:
```suggestion
		height: $clickable-area;
```

Same for the `line-height` line 56 btw